.storybook/manager.js

```diff
@@ -33,10 +36,9 @@ addons.setConfig({
 		textColor: "rgb(34, 34, 34)",
 
 		// Toolbar default and active colors
-		barTextColor:
-			"rgb(34, 34, 34)" /* --spectrum-neutral-content-color-default */,
-		barSelectedColor: "rgb(2, 101, 220)" /* --spectrum-global-color-blue-900 */,
-		// barBg: 'rgb(248, 248, 248)', /* --spectrum-alias-background-color-default */
+		barTextColor: "rgb(34, 34, 34)",
+		barSelectedColor: "rgb(2, 101, 220)",
+		// barBg: 'rgb(248, 248, 248)',
```

Comment:
✨ Should we remove this if it's no being used?

Reply:
I would but these values aren't well documented so I didn't want to forget that this keyword did something in the future.